manifests/0000_50_cluster-monitoring-operator_04-config.yaml

```diff
@@ -240,6 +240,11 @@ data:
     #
     # owners: (https://github.com/kubevirt)
     #
+    # cnv_abnormal holds issues with the pods for each container, e.g. memory exceeded value.
```

Comment:
@avlitman 
    # cnv_abnormal represents the reason why the operator might have an issue
    # and includes the node, namespace, container, reason labels.

Reply:
fixed